backend/db/V2_2__change_user_data_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE brevity.user ALTER COLUMN id TYPE varchar(25), ALTER COLUMN email TYPE varchar(254);
```

Comment:
why not just put this at the end of the previous one since theyre both new versions in this pr anyways?

Reply:
yup will do